build.gradle

```diff
@@ -0,0 +1,2 @@
+ext.dockerImageName = 'catalog_kafkatrigger'
```

Comment:
I think I just patterned it after the cloudant image name.  I can name it whatever you want it to be named.